benchmarks/benchmarks.rb

```diff
@@ -2,13 +2,17 @@
 
 require "benchmark/ips"
 
-require "memery"
 require "memo_wise"
-require "memoist"
-require "memoized"
-require "memoized/version" # Memoized::VERSION does not get loaded above.
-require "memoizer"
-require "memoizer/version" # Memoizer::VERSION does not get loaded above.
+
+# Some gems do not yet work in Ruby 3 so we only require them if they're loaded
+# in the Gemfile.
+%w[memery memoist].each { |gem| require gem if Gem.loaded_specs.key?(gem) }
```

Comment:
No strong preference here either, but just looking again, could we include `memoized` and `memoizer` in the first line to make the second line also a one liner?
```
%w[memery memoist memoized memoizer].each { |gem| require gem if Gem.loaded_specs.key?(gem) }
# The VERSION constant does not get loaded above. 
%w[memoized memoizer].each { |gem| require "#{gem}/version"  if Gem.loaded_specs.key?(gem) }
```

Reply:
Nice suggestion! Unfortunately those lines are too long. The best we could do would be:

```ruby
# Some gems do not yet work in Ruby 3 so we only require them if they're loaded
# in the Gemfile.
%w[memery memoist memoized memoizer].
  each { |gem| require gem if Gem.loaded_specs.key?(gem) }

# The VERSION constant does not get loaded above.
%w[memoized memoizer].
  each { |gem| require "#{gem}/version" if Gem.loaded_specs.key?(gem) }
```

Up to you; I'm happy with whichever you prefer.

Reply:
I think I like this better than what we have, if you don't mind changing it! Reads cleaner to me